### ISSUE TITLE
Has beutiful json string that has key and value different color

### ISSUE THREAD

**Issue by hafiz013** [FasterXML/jackson-databind#3502]
If this library possible in the future make key and value of json beauty different color like in apps postman.

**Comment by yawkat** [comment]
jackson is not responsible for syntax highlighting of output.

**Comment by hafiz013** [comment]
I just image if can do it.

**Comment by skjolber** [comment]
@yawkat see [jackson-syntax-highlight](https://github.com/skjolber/jackson-syntax-highlight) (usage: [example1](https://github.com/skjolber/logback-logstash-syntax-highlighting-decorators) -> [example2](https://github.com/entur/cloud-logging/blob/8f6bc2d8ec585cf1bee3fa78e16b35ed4985db28/gcp/spring-boot-autoconfigure-gcp-test/src/main/resources/logback/logback-spring-test.xml#L25)). 

@cowtowncoder not quite sure what you meant by jackson module, but if the above library could serve as a starting point for a module under the jackson (i.e. fasterxml) umbrella, then two thumbs up from me - actually just tried the latest 3.x release candidate and hit some bumps.

Edit: Filed an issue on 3.x: https://github.com/FasterXML/jackson-databind/issues/5331

**Comment by cowtowncoder** [comment]
Right, I am not 100% sure how this could be packaged as a `[Jackson]Module` (or Format backend).

Problem wrt Jackson 3.x is immutability of constructed `TokenStreamFactory`. Well that and also there is no way to configure a TSF (like `JsonFactory`) to wrap `JsonGenerator` automatically (having ability to do that seems like a useful extension, but one doesn't exist as of now).

So for now I think what could be done would probably be just a link from https://github.com/FasterXML/jackson/ (`README.md`) to this project.
